docs/documentation/upgrading/topics/changes/changes-25_0_0.adoc

```diff
@@ -116,6 +117,17 @@ The reason for this change is that the value of the `sub` claim in the refresh t
 
 However a `refresh_token_sub` detail is now added as backwards compatibility to have info about the user in the case of missing userId in the `REFRESH_TOKEN_ERROR` event.
 
+= Using older javascript adapter
+
+In case that you use latest {project_name} server with older versions of javascript adapter in your applications, you may be affected by the token changes mentioned above as previous
+versions of javascript adapter rely on the claims, which were added by {project_name}, but not supported by the OIDC specification. This include:
+
+* Adding `Session State (session_state)` mapper in case of using Keycloak Javascript adapter 24.0.3 or older
+* Adding `Nonce backwards compatible` mapper in case of using Keycloak Javascript adapter older than Keycloak 24
```

Comment:
```suggestion
* Adding the `Session State (session_state)` mapper in case of using Keycloak Javascript adapter 24.0.3 or older
* Adding the `Nonce backwards compatible` mapper in case of using a Keycloak Javascript adapter that is older than Keycloak 24
```